entrypoint.sh

```diff
@@ -150,5 +157,12 @@ set_download_url
 check_version_exists
 download_aws_cli
 install_aws_cli
+if [[ $_LIGHTSAIL_INSTALL = "true" ]]; then
```

Comment:
please move this logic inside the function that you've created `install_lightsailctl`

Reply:
Initially I thought that it better to wrap it since the function wouldn't always install but no, consistency trumps it!